### ISSUE TITLE
Could not load package metadata for package modx.transport.

### ISSUE THREAD

**Issue by Ibochkarev** [modxcms/revolution#15450]
### Summary
Error in the error log Could not load package metadata for package modx.transport. Upgrade your model.

### Step to reproduce
Set `log_level` to 4.
Open Extras -> Installer

Open MODX error log.

It should be clear.

### Observed behavior

https://user-images.githubusercontent.com/2138260/110108953-088f9400-7dd7-11eb-87a9-e369179f5ee9.mp4

``` bash
[2021-03-05 11:36:37] (WARN @ /home/s22270/www/core/vendor/xpdo/xpdo/src/xPDO/xPDO.php : 526) Could not load package metadata for package modx.transport. Upgrade your model.
```

### Related issue(s)/PR(s)
NA

### Environment
MODX Revolution 3.0.0-alpha3 (git), PHP Version 7.4.12

**Comment by Jako** [comment]
I am working on this.

**Comment by Jako** [comment]
I can't reproduce this.

**Comment by Ibochkarev** [comment]
@Jako Sorry, I made a typo - `log_level` there should be 4

**Comment by Ibochkarev** [comment]
Fixed #15466
